frontend/ui-components.js

```diff
@@ -0,0 +1,93 @@
+import { signInWithNearWallet, signOutNearWallet } from './near-api';
+import { getConfig } from './config';
+
+export function SignInPrompt() {
+  return (
+    <main>
+      <h1>
+        Welcome to NEAR!
+      </h1>
+      <p>
+        Your contract stores information on the NEAR blockchain. To
+        start you need to sign in using the NEAR Wallet. It is very simple,
+        just use the button below.
+      </p>
+      <p>
+        Do not worry, this app runs in the test network ("testnet"). It works
+        just like the main network ("mainnet"), but using NEAR Tokens that are
+        only for testing!
+      </p>
+      <br/>
+      <p style={{ textAlign: 'center' }}>
+        <button onClick={signInWithNearWallet}>Sign in with NEAR Wallet</button>
+      </p>
+    </main>
+  );
+}
+
+export function SignOutButton() {
+  return (
+    <button className="link" style={{ float: 'right' }} onClick={signOutNearWallet}>
+      Sign out
+    </button>
+  );
+}
+
+export function NearInformation({ message }) {
+  const { networkId, contractName } = getConfig(process.env.NODE_ENV || 'development')
```

Comment:
```suggestion
  const { networkId, contractName } = getConfig(process.env.NODE_ENV || 'testnet')
```
we actually deprecated the 'development' network, this is a artifact from `create-near-app`